src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/contactperson/ContactPersonDomainEventListener.kt

```diff
@@ -33,16 +33,38 @@ class ContactPersonDomainEventListener(
   ): CompletableFuture<Void?> = onDomainEvent(rawMessage) { eventType, message ->
     when (eventType) {
       "contacts-api.contact.created" -> contactPersonService.contactCreated(message.fromJson())
-      "contacts-api.prisoner-contact.create" -> log.info(eventType)
+      "contacts-api.prisoner-contact.created" -> contactPersonService.prisonerContactCreated(message.fromJson())
       else -> log.info("Received a message I wasn't expecting: {}", eventType)
     }
   }
 }
 
```

Comment:
All contact events will have a "source" attribute - interface stuff so this is generic